src/client/components/vision2/styles.css

```diff
@@ -0,0 +1,14 @@
+.vision {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  flex-shrink: 1;
+  text-align: center;
+}
+
+.cameras {
```

Comment:
> The canvas should maintain the aspect ratio of the image if possible

It does! The black bars are just from a css background color.

> Also, it would be good to be able to see more than a single image at a time.

Indeed. I think a some reasonable camera layout algorithm is a prerequisite to merging this PR.